packages/webpack-cli/__tests__/arg-parser.test.js

```diff
@@ -96,6 +96,16 @@ const basicOptions = [
         multiple: true,
         description: 'multi flag',
     },
+    {
+        name: 'processor-flag',
+        usage: '--processor-flag',
+        type: Boolean,
+        description: 'flag with processor',
+        processor(opts) {
+            opts.processed = opts.processorFlag;
+            delete opts.processorFlag;
```

Comment:
Can we do ?

```suggestion
        processor: (opts) => {
            opts.processed = opts.processorFlag;
            delete opts.processorFlag;
```

Reply:
Why?

Reply:
just to match like other properties.

Reply:
ArrowFunctionExpression and MethodDefinition is different thing and can used for different purposes